src/Compilers/CSharp/Portable/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -6102,8 +6102,8 @@ internal AnonymousMethodExpressionSyntax(SyntaxKind kind, SyntaxToken asyncKeywo
     /// </summary>
     public override BlockSyntax Block => this.block;
     /// <summary>
-    /// Inherited from AnonymousFunctionExpressionSyntax, but not used for 
-    /// AnonymousMethodExpressionSyntax.  Thsi will always be null.
+    /// Inherited from AnonymousFunctionExpressionSyntax, but not used for
+    /// AnonymousMethodExpressionSyntax.  This will always be null.
```

Comment:
Note: no actual code changes in the .cs files that compile into roslyn.  It's just the fixups of some comments.  So this is really just a change to Syntax.xml not the Roslyn binaries.